### PR TITLE
New bugfixes, color changes, and more.

### PR DESCRIPTION
**Pull request by Ajay9o9**
1. Updated .gitignore file with addition of **package-lock.json**
2. Changed main theme color from black to **gray**. 
3. Few other minor bug fixes

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -6,5 +6,5 @@
 body {
   margin: 0;
   padding: 0;
-  background-color: black;
```
Comment (sindhuhinesh) [inline src/App.css:9]: why did you change the color?

Comment (Ajay9o9) [reply src/App.css:9]: Its as per the new UI/UX design changes.
